internal/testrunner/runners/system/runner.go

```diff
@@ -328,12 +330,20 @@ func (r *runner) runTest(config *testConfig, ctxt servicedeployer.ServiceContext
 	// Reload test config with ctx variable substitution.
 	config, err = newConfig(config.Path, ctxt, serviceOptions.Variant)
 	if err != nil {
-		return result.WithError(errors.Wrap(err, "unable to reload system test case configuration"))
+		return result.WithError(fmt.Errorf("unable to reload system test case configuration: %w", err))
 	}
 
 	kib, err := kibana.NewClient()
 	if err != nil {
-		return result.WithError(errors.Wrap(err, "can't create Kibana client"))
+		return result.WithError(fmt.Errorf("can't create Kibana client: %w", err))
+	}
+
+	// Install the package before creating the policy, so we control exactly what is being
+	// installed.
+	logger.Debug("Installing package...")
+	err = installPackage(kib, pkgManifest, r.options.PackageRootPath)
+	if err != nil {
+		return result.WithError(fmt.Errorf("unable to install package: %w", err))
 	}
```

Comment:
Should it be uninstalled the package using the kibana client at the end of the test run too?

Reply:
Added. Good point.